src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -717,6 +720,13 @@ class VAMap extends Component {
   };
 
   render() {
+    const chatbotLink = this.props.showCovidChatbotLink && (
+      <>
+        For answers to questions about how COVID-19 may affect your VA health
+        appointments, benefits, and services, use our VA{' '}
+        <a href="/coronavirus-chatbot/">coronavirus chatbot</a>.
+      </>
```

Comment:
Should this be its own paragraph?

Reply:
Not according to the copy in [this ticket](https://github.com/department-of-veterans-affairs/covid19-chatbot/issues/121#issue-603391159).